stable/newrelic-infrastructure/templates/daemonset.yaml

```diff
@@ -32,21 +32,28 @@ spec:
           securityContext:
             privileged: true
           env:
+            {{- if or .Values.licenseKey .Values.customSecret }}
```

Comment:
This conditional here makes no sense to me. Would it be possible to mark these values as mutually required?

Reply:
We could try to do something with the `required` function: https://github.com/helm/helm/blob/master/docs/charts_tips_and_tricks.md#know-your-template-functions

Reply:
I think you are right, if any of these values is not set, daemon set will not be created at all - will fix it.